doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='sso-remote-services-securing-proc']
-= Securing {CENTRAL} file system services using RH-SSO
+= Enabling access to external file system and Git repository services for {CENTRAL} using RH-SSO
 
-To consume other remote services such as file systems (for example, a remote GIT service), you must specify the correct RH-SSO login module.
+To enable the {CENTRAL} to consume other remote services such as file systems and Git repositories using RH-SSO authentication, you must create a configuration file.
```

Comment:
[Style] I think adding a comma as below will help with the long sentence or perhaps the middle content can be separated?

To enable the {CENTRAL} to consume other remote services, such as file systems and Git repositories using RH-SSO authentication, you must create a configuration file.